configuration/configurator/BuildSet.hpp

```diff
@@ -0,0 +1,122 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2015 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+#ifndef _BUILD_SET_HPP_
+#define _BUILD_SET_HPP_
+
+#include "jstring.hpp"
+#include "jlib.hpp"
+
+namespace CONFIGURATOR
+{
+
+static const char* DEFAULT_BUILD_SET_XML_FILE("buildset.xml");
+static const char* DEFAULT_BUILD_SET_DIRECTORY("/opt/HPCCSystems/componentfiles/configxml/");
```

Comment:
I will grab the path from build-config.h.  This code does not use esp code/libraries, and it is not required to run in an esp, e.g. it can run as a command line executable.